pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform/project.json

```diff
@@ -1,5 +1,6 @@
 {
   "dependencies": {
+    "Microsoft.Net.Native.Compiler": "1.5.0",
```

Comment:
The intention is to update the metapackage version to 5.2.3 for this change. Are you saying that will happen automatically? Or do we need to do another process to update the version?